dataikuapi/dss/ml.py

```diff
@@ -722,6 +722,86 @@ def get_partial_dependencies(self):
         return DSSPartialDependencies(data)
 
 
+class DSSSubpopulationModality(DSSExtendableDict):
+    """
+    Object to read details of a subpopulation analysis modality
+
+    Do not create this object directly, use :meth:`DSSSubpopulationAnalysis.get_modality(definition)` instead
+    """
+
+    def __init__(self, computed_as_type, data):
+        super(DSSSubpopulationModality, self).__init__(data)
+
+        if computed_as_type == "CATEGORY":
+            self.definition = DSSSubpopulationCategoryModalityDefinition(data)
+        elif computed_as_type == "NUMERIC":
+            self.definition = DSSSubpopulationNumericModalityDefinition(data)
+    
+    def get_raw(self):
+        """
+        Gets the raw dictionary of the subpopulation analysis modality
+        """
+        return self.internal_dict
+    
+    def get_definition(self):
+        """
+        Gets the definition of the subpopulation analysis modality
+
+        :returns: definition
+        :rtype: :class:`dataikuapi.dss.ml.DSSSubpopulationModalityDefinition`
+        """
+        return self.definition
+    
+    def excluded(self):
```

Comment:
Consider a clearer version:

```suggestion
    def is_excluded(self):
```

How is it done elsewhere?